HiP-UserStore/appsettings.json

```diff
@@ -22,11 +22,11 @@
         }
     },
 
-    "Endpoints": {
-        "MongoDbHost": "mongodb://localhost:27017",
-        "MongoDbName": "main",
-        "ThumbnailUrlPattern": null
-    },
+  "Endpoints": {
+    "MongoDbHost": "mongodb://localhost:27017",
+    "MongoDbName": "main",
+    "ThumbnailUrlPattern": null   
```

Comment:
Since we are now using the DataStore SDK, there should also be an entry for the DataStoreHost with default value null (see Achievements).